tailwind.config.js

```diff
@@ -61,6 +52,17 @@ module.exports = {
 			l: '24px',
 			full: '9999px',
 		},
+		extend: {
```

Comment:
@malinovic Please note that I have expanded the Tailwind spacing. This allows me to use 12px for medium button padding. A different and maybe better approach would have been to add '12px' to the custom spacings. However, this would probably involve a rewriting classes that have already been applied in existing components.